framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -180,8 +180,13 @@ VulkanReplayConsumerBase::VulkanReplayConsumerBase(std::shared_ptr<application::
             break;
         default:
             swapchain_ = std::make_unique<VulkanVirtualSwapchain>();
-            swapchain_->SetPerformanceMode(options.virtual_swapchain_skip_blit);
     }
+
+    VulkanSwapchainOptions swapchain_options;
+    swapchain_options.skip_additional_present_blts = options.virtual_swapchain_skip_blit;
```

Comment:
Could `options` be renamed `replay_options`?  I had trouble reading these two lines because `options_` is similar to to `options`.

Reply:
`options_` is the global class member version of `options`.  And both `options` and `options_` are variables of type `VulkanReplayOptions`.  So I can rename just this one function's `options` to `replay_options` but `replay_options` will then be used to initialize `options_`.  And I don't want to globally change `options_` to `replay_options_`  since then it's going to flow into many other files. 

That all being said, I think it might just be better to leave it as is.  What do you think @bradgrantham-lunarg ?